corehq/apps/app_manager/static/app_manager/js/app-notifications.js

```diff
@@ -0,0 +1,14 @@
+/* globals hqDefine moment alert_user */
+hqDefine('app_manager/js/app-notifications.js', function () {
+    function NotifyFunction(userId) {
```

Comment:
that is pretty neat, though I'm not sure an out-of-browser experience is actually superior in this case since you might have a ton of things open and want to know what it was referring to. do you mean in addition to the page-level notification?